README.md

```diff
@@ -77,6 +77,18 @@
 
    To run migrations locally, use `yarn migrate` from the root. If you ever want to replace your db contents with a fresh setup, run `yarn db:fresh`.
 
+   If you need to make a new migration, simply run `yarn workspace @hangar/database migration:create`. There are also scripts for running the `up` and `down` commands: `migration:up` and `migration:down` respectively.
+
```

Comment:
it would be pretty cool to have a reference to link to when alerting the team of migration changes since they are so sparse.
```suggestion

   If you need to incorporate a new migration from a recently merged feature branch, run `yarn workspace @hangar/database migration:down` and ` yarn db:seed`. `seed` will both perform the `migration:up` action, and provide any new data seeding that might compliment the migration.

```